src/style.css

```diff
@@ -0,0 +1,55 @@
+body {
+  margin: 0;
+}
+
+.container {
+  height: 96px;
+  width: 96px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_star {
+  background-image: url(images/star.svg);
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-left: 4px;
+}
+
+.stars--1 .stars_star:first-child {
+  background-image: url(images/star-active.svg);
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-left: 4px;
+}
+
+.stars--2 .stars_star:first-child,
+.stars--2 .stars_star:nth-child(2),
+.stars--3 .stars_star:first-child,
+.stars--3 .stars_star:nth-child(2),
+.stars--3 .stars_star:nth-child(3),
+.stars--4 .stars_star:first-child,
+.stars--4 .stars_star:nth-child(2),
+.stars--4 .stars_star:nth-child(3),
+.stars--4 .stars_star:nth-child(4),
+.stars--5 .stars_star:first-child,
+.stars--5 .stars_star:nth-child(2),
+.stars--5 .stars_star:nth-child(3),
+.stars--5 .stars_star:nth-child(4),
+.stars--5 .stars_star:nth-child(5) {
```

Comment:
try to check the `:nth-child` pseudo-class to reduce the code. here can be used next construction as `:nth-child(-n + 3)` etc.
here is the [doc](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child)